### DISCUSSION TITLE
Include achievements from private repos.

### DISCUSSION THREAD

**Discussion by muzam1l** [community/community#18372] [Other Feature Feedback, Questions, & Ideas] [answered]
It looks lame to have these new achievements stats from public repos only. There can be display of achievement tags without mentioning repo or pr for private repos.

**Comment by invisible-defects** [comment]
It seems to me that it doesn't count from public repos only - I have achievements from private repos. However, those repos were collaborative and had 3+ contributors

**Comment by muzam1l** [comment, marked as answer]
Better described in #18243
